### DISCUSSION TITLE
tb-profiler products

### DISCUSSION THREAD

**Discussion by justicengom** [jodyphelan/TBProfiler#182] [General]
Please I would like to know which type of file from tb-profiler product that I can use for visualizing to Itol.
Thank you

**Comment by jodyphelan** [comment]
Hi @justicengom 

After you run `tb-profiler collate` you will see a few files with the extension `.itol.txt`. You can use them as config files for your trees.

**Reply by justicengom** [reply]
thank you

**Comment by justicengom** [comment]
Hi Jody
Thank you very much for your answer
